api/controllers/PublicController.js

```diff
@@ -59,7 +59,7 @@ module.exports = {
           throw new Error('Email not recognised.')
         } else {
           randomString = ForgotPass.randomString()
-          return Members.update({ id: member.id }, { new_password: randomString })
+          return Members.update({ id: member.id }, { new_password: randomString, id: member.id })
```

Comment:
That doesn't make sense to me, because where you've added the id is only used to update rather than to find. Or so I thought ...